index.html

```diff
@@ -14,12 +28,12 @@
       href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
       rel="stylesheet"
     />
-    <title>Osuagwu Lucky James</title>
+    <title>My Portfolio Website</title>
```

Comment:
- [ ] It is better to have a better & meaningful page title such as Osuagwa's Portfolio. Make this change